scripts/DMF/compile

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
Ok for PoC, sometime should become part of a Makefile... like other scriptlets in this dir ;)